### PR TITLE
HRIS-161 [BE/FE] Manager Overtime Management Rework

### PR DESCRIPTION
**Pull request by NiloJr-sun**
## Issue Link

https://framgiaph.backlog.com/view/HRIS-161

## Definition of Done

- [x] A manager can view all overtime request.
- [x] A manager can approve or disapprove requests regardless of the current status.

## Notes
- N/A

## Pre-condition
- In the root directory ``` docker compose up --build ```

## Expected Output
- In manager position view, it should reflect all the overtime requests

## Screenshots/Recordings

https://user-images.githubusercontent.com/109492180/227508696-24848b52-ef29-4e9c-80c3-33679905b4b5.mp4

### CODE REVIEW

**Review by arden-dave — APPROVED** [review]
Approve. 👍

File: client/src/components/molecules/OvertimeManagementTable/columns.tsx [hunk 1]
```diff
@@ -419,9 +422,17 @@ export const managerColumns = [
         })
       }
 
+      const isManager = (): boolean => {
+        if (positionId === Position.MANAGER) {
+          return true
+        } else {
+          return false
+        }
```
Comment (abduljalilpalala) [inline client/src/components/molecules/OvertimeManagementTable/columns.tsx:430]:
```suggestion
        return (positionId === Position.MANAGER)
```

File: client/src/components/molecules/OvertimeManagementTable/columns.tsx [hunk 2]
```diff
@@ -442,9 +453,32 @@ export const managerColumns = [
                 </Button>
               </Tippy>
             </>
+          ) : (
+            props.row.original.isManagerApproved == null && (
+              <>
```
Comment (abduljalilpalala) [inline client/src/components/molecules/OvertimeManagementTable/columns.tsx:458]: Destruct the props object

File: client/src/components/molecules/OvertimeManagementTable/columns.tsx [hunk 3]
```diff
@@ -442,9 +453,32 @@ export const managerColumns = [
...
+                    <X className="h-4 w-4 stroke-[3px]" />
+                  </Button>
+                </Tippy>
+              </>
+            )
           )}
 
-          {props.row.original.isManagerApproved && (
+          {!isManager() && props.row.original.isManagerApproved && (
             <>
```
Comment (abduljalilpalala) [inline client/src/components/molecules/OvertimeManagementTable/columns.tsx:482]: Destruct the props object.

**Review by JohnRoyCabezas — APPROVED** [review]

**Pull request merged**